kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/HasMetadataOperation.java

```diff
@@ -78,8 +78,10 @@ public T replace(T item) {
         if (got == null) {
           return null;
         }
+
+        item = handleReplace(item);
```

Comment:
Haha sorry it's badly worded (it's late here!). This is definitely an issue & I'll take a look asap at fixing.